src/test/java/org/cactoos/io/GzipOutputTest.java

```diff
@@ -82,8 +82,8 @@ public void writeToGzipOutput() throws Exception {
 
     @Test(expected = IOException.class)
     public void writeToClosedGzipOutput() throws Exception {
-        final OutputStream stream = new FileOutputStream(
-            this.folder.newFile("cactoos.txt")
+        final OutputStream stream = Files.newOutputStream(
```

Comment:
@ilyakharlamov please, check if is possible another kind of object instead of use a static method.

Reply:
fixed